__tests__/app.test.js

```diff
@@ -3,18 +3,16 @@ const app = require("../app.js");
 const db = require("../db/connection.js");
 const seed = require("../db/seeds/seed.js");
 const testData = require("../db/data/test-data/index.js");
+const endpoints = require("../endpoints.json");
 
 beforeEach(() => {
   return seed(testData);
 });
 afterAll(() => {
-  db.end();
+  return db.end();
 });
 
 describe("GET /api/topics", () => {
-  test("should respond with a 200 status code", () => {
-    return request(app).get("/api/topics").expect(200);
-  });
   test("should respond with a 404 status code with invalid endpoint", () => {
     return request(app).get("/api/topix").expect(404);
```

Comment:
This test will always pass as the 404 has been sent by express. If you also expect the body to have a msg: 'Not Found!' then this test will fail and would require us to handle the error ourselves at the end of the express chain in app.js